Package.swift

```diff
@@ -21,7 +21,7 @@ import PackageDescription
 let package = Package(
 	name: "Stagehand",
 	platforms: [
-		.iOS(.v12),
+		.iOS(.v15),
 		.macOS(.v11),
 	],
 	products: [
```

Comment:
Do we want the swift version of this repo to be 5 or 6? I know we want to support SPM v6, but what language mode do we want for this repo? Are we ready for 6? Or is there more work to do first?